pysat/_instrument.py

```diff
@@ -1985,14 +1984,11 @@ def _filter_netcdf4_metadata(self, mdata_dict, coltype, remove=False,
         for key in mdata_dict:
             if type(mdata_dict[key]) == bool:
                 mdata_dict[key] = int(mdata_dict[key])
-        # Should use isinstance here
-        if (coltype == type(' ')) or (coltype == type(u' ')):
-            # if isinstance(coltype, str):
+        if (coltype == str):
```

Comment:
Not sure if this is a remnant from python 2.7.  If I run `type(' ')` and `type(u' ')` in ipython, both are returned as `str`.

Reply:
It is a remanent of python 2.7